test/Test/Smoke.hs

```diff
@@ -106,7 +108,7 @@ testUpgradeToV1 = TestUpgrade $ \admin redeem balances tzbtc -> do
       , upNewCode = tzbtcContractRouterV1
       , upNewPermCode = emptyPermanentImpl
       }
-  transferMoney admin $ toMutez 50 * 1000000 -- 50 XTZ
+  transferMoney admin [tz|50|] -- 50 XTZ
```

Comment:
Perhaps, it's worth trying to mark `admin` address as refillable and remove these hard-coded transfer amounts. AFAIR, the main goal of these transfers is to provide `admin` enough money for origination and upgrade

Reply:
I would argue that should be a separate PR.

Reply:
Right, I've even opened it some time ago https://github.com/tz-wrapped/tezos-btc/issues/173